x-pack/filebeat/module/zoom/webhook/config/webhook.yml

```diff
@@ -0,0 +1,35 @@
+{{ if eq .input "http_endpoint" }}
+
+type: http_endpoint
+listen_address: {{ .listen_address }}
+listen_port: {{ .listen_port }}
+prefix: {{ .prefix }}
+basic_auth: {{ .basic_auth }}
+username: {{ .username }}
+username: {{ .password }}
+content_type: "{{ .content_type }}"
+secret: {{ .secret | tojson }}
```

Comment:
This should allow users to configure the `ssl.*` options. Perhaps like `ssl: {{ .ssl | tojson }}`?